module/storage-elasticsearch-aws/src/test/java/zipkin/module/storage/elasticsearch/aws/ZipkinElasticsearchAwsStorageModuleTest.java

```diff
@@ -106,13 +95,14 @@ void expectSignatureInterceptor() {
     Consumer<ClientOptionsBuilder> customizer =
         (Consumer<ClientOptionsBuilder>) context.getBean("awsSignatureVersion4", Consumer.class);
 
-    ClientOptionsBuilder clientBuilder = new ClientOptionsBuilder();
-
-    // TODO(anuraaga): Can be simpler after https://github.com/line/armeria/issues/1883
+    ClientOptionsBuilder clientBuilder = ClientOptions.builder();
     customizer.accept(clientBuilder);
-    Client<HttpRequest, HttpResponse> decorated = clientBuilder.build().decoration()
-        .decorate(HttpRequest.class, HttpResponse.class, mockHttpClient);
-    assertThat(decorated).isInstanceOf(AWSSignatureVersion4.class);
+
+    WebClient client = WebClient.builder("http://127.0.0.1:1234")
+        .option(ClientOption.DECORATION, clientBuilder.build().decoration())
```

Comment:
Think it's more precise to do `.options(clientBuilder.build())`

Reply:
excellent I like less lines and imports for the same job